core/ui/card_ui/settings/general_controller_settings_menu.gd

```diff
@@ -1,5 +1,42 @@
 extends Control
 
+var input_plumber := load("res://core/systems/input/input_plumber.tres") as InputPlumberInstance
+var state_machine := load("res://assets/state/state_machines/menu_state_machine.tres") as StateMachine
+var state := load("res://assets/state/states/gamepad_settings.tres") as State
+
+@onready var gamepad_button := $%GamepadSettingsButton as CardButton
+
 # Called when the node enters the scene tree for the first time.
 func _ready():
-	pass
+	gamepad_button.gui_input.connect(_on_button_input)
+
+
+# If the user presses the gamepad settings button with a non-gamepad input method,
+# open the settings for the first detected controller.
+# TODO: Add a controller select pop-up to select a gamepad to configure
```

Comment:
We should prioritize this soon, along with gamepad ordering. This shouldn't be exclusive to mouse/touch input though, I think it would be reasonable to configure multiple gamepads with player 1 for example .